src/static/store/modules/tests/types.ts

```diff
@@ -5,6 +5,9 @@ export interface TestsStore {
   skips: Suite[];
   stats: Stats;
   gui: boolean;
+  suites?: any;
```

Comment:
Type declaration of 'any' loses type-safety. Consider replacing it with a more precise type.